### ISSUE TITLE
Users

### ISSUE THREAD

**Issue by nietsymerej** [nietsymerej/collecobrary#8]
We need to get some users and we need to get some feedback.

Any ideas?

**Comment by BenutzerEinsZweiDrei** [comment]
Personal Opinion on Feedback: i dont know how the graph is adding any value to the app.

Like its good to have some design element somewhere. That makes fun.

And i guess it was insane work creating this thing.

But basically i would replace it with a more visually appealing visualisation of the learning path.

For example by googeling i found this

https://fifty1.com/wp-content/uploads/2023/03/Blog-fifty1-learning-journey-980x653.jpg

its one path or one line in any direction on the page.

And from this line there is going bubbles in all directions as steps of the learning path.

Those bubbles can for example have just two letters to the course. Later you could add svg symbols.

When you click the bubble the information to the course pops up. With an alternernarive tree view of the course structure.

**Comment by IlIllII** [comment]
I see what you're saying. The network is a bit chaotic and not super visually appealing.

I found it challenging to create systematic learning paths, even for subjects I was familiar with. It required a lot of editorial decisions about ordering, which are hard to do in general, but were impossible for subjects I didn't know. I thought it more tenable to use course prerequisites to structure the graph, which also made it easier to add new courses without restructuring the visualization every time.

I was aiming for something like this:

![Screenshot 2024-10-31 at 8 18 08 AM](https://github.com/user-attachments/assets/1acb94ce-56bc-4dfc-855b-d6f3ed34783b)

Do you think the graph is useless? If you have concrete suggestions I can make the changes, or you can explore the codebase and see if you can come up with something more compelling. If you do, we can either replace the graph or create a toggle for different kinds of visualizations.

**Comment by BenutzerEinsZweiDrei** [comment]
I need to first figure out vue and deployment on github pages... Guess i will need some time 🥹🥹

Can i come to discord for assistance?

**Comment by IlIllII** [comment]
Sorry, I should probably remove that - I don't think it's been active in a while, but I haven't checked recently! If you want to ask questions here I'll do my best to answer them.

Edit: Looks like I did remove the discord, so for now here is the only place to collaborate

**Comment by BenutzerEinsZweiDrei** [comment]
I get it that basically the course structure is like the evidence that it makes sense to study the topic in that order. So its the argument for the quality of the course. So it needs to be included and understood.

My idea is just to somehow communicate this information more intuitevly so its understood on the go as you are using the page..

**Comment by BenutzerEinsZweiDrei** [comment]
Sorry created this on the phone in a meme app but i hope you get the sentiment 

The idea behind this design is to have a dashboard 

- where you navigate the course contents (for example clicking a color filters all the contents to those parts of the course and clicking a course shows its list of prerequistes)
- and by navigating you understand the course structure more intuitevly 
- and then in the next step you use your dashboard to keep track of your progress

Every click in 
- course list
- legend
- course structure
- prerequesits

Filters all the contents of the dashboard to put the focus on the clicked course and show its prerequesites (and maybe even add follow up courses)

For example just add opacity to all other contents so clicked course is in focus
![IMG_20241031_175319](https://github.com/user-attachments/assets/479c1435-2a11-4fc8-a68c-dc103881bcfa)

**Comment by IlIllII** [comment]
Mmm I like the prerequisites box idea!

Regarding the dashboard, I am not following exactly. Could you explain more of what you're suggesting or maybe show a demo?

**Comment by BenutzerEinsZweiDrei** [comment]
yes i will try to work my way into it and figure out how to deploy on pages then i hopefully can create ideas to show
